clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/DownloadUtil.ts

```diff
@@ -182,7 +182,15 @@ export class DownloadUtil {
     return action === UNLOCK_DOWNLOAD_QUEUE_MSG;
   }
 
-  private static isInvalidCredentials(e: any) {
-    return e?.mDetails?.errorCode === 401;
+  private static isInvalidCredentials(e: unknown) {
+    return (
+      e &&
+      typeof e === "object" &&
+      "mDetails" in e &&
```

Comment:
is possible to write a `hasMember` function? Something like
```
function hasMember<U, T>(e: U, m: T): T is keyof U
```
?

Reply:
Good idea 👍 Added ✅